gateway/gateway.go

```diff
@@ -39,12 +39,12 @@ var defaultQueryFrontendAPIs = []string{
 }
 
 func New(config *Config, srv *server.Server) (*Gateway, error) {
-	distributor, err := NewProxy(config.Distributor.URL)
+	distributor, err := NewProxy(config.Distributor.URL, config.Distributor.Timeouts, DISTRIBUTOR)
 	if err != nil {
 		return nil, err
 	}
 
-	frontend, err := NewProxy(config.QueryFrontend.URL)
+	frontend, err := NewProxy(config.QueryFrontend.URL, Timeouts{}, "")
```

Comment:
Since this is just to make sure I am doing the right thing, I passed empty values to this function. I will fix it as soon as I get an approval